src/features/edit/js/gridEdit.js

```diff
@@ -112,12 +112,45 @@
               }
             },
             methods: {
-              edit: { }
+              edit: {
+                /**
+                 * @ngDoc method
```

Comment:
Sorry nube to this framework. so in the context of this framework what is the definition of a method?
In my understanding a function is not a property of an object.